Scan/MatrixARM/matrix_scan.c

```diff
@@ -403,6 +426,101 @@ void Matrix_scan( uint16_t scanNum )
 		Matrix_pin( Matrix_cols[ strobe ], Type_StrobeOff );
 	}
 
+
+	// Matrix ghosting check and elimination
+	// . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . 
+#ifdef GHOST
+	// strobe = column, sense = row
+
+	// Count (rows) use for columns
+	//print("C ");
```

Comment:
If useful, I'd recommend adding debug code to check for matrixDebugMode so you can re-enable later. Otherwise, you can leave the debug code commented out.